internal/kernel_neon.h

```diff
@@ -727,7 +985,8 @@ struct NEON_64bit_GEMM_Int8Operands_LhsNonzero : KernelBase {
         "beq " GEMMLOWP_LABEL_AFTER_LOOP_LAST16 "f\n"
 
         // General loop.
-        GEMMLOWP_LABEL_LOOP ":\n"
```

Comment:
That makes sense, I was just confused because I thought it was all clang formatted already so the change surprised me. LGTM.